hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/format/mor/MergeOnReadInputFormat.java

```diff
@@ -753,7 +761,9 @@ private Option<IndexedRecord> mergeRowWithLog(
         String curKey) throws IOException {
       final HoodieAvroRecord<?> record = (HoodieAvroRecord) scanner.getRecords().get(curKey);
       GenericRecord historyAvroRecord = (GenericRecord) rowDataToAvroConverter.convert(tableSchema, curRow);
-      return record.getData().combineAndGetUpdateValue(historyAvroRecord, tableSchema);
+      // TODO IndexedRecord to HoodieRecord
```

Comment:
is this TODO part of step 2 work?

Reply:
At that time, `TODO` was because `HoodieFlinkRecord` was not implemented. Currently, Avro record is still used. 
I will remove this `TODO` first, and wait for the later implementation of `HoodieFlinkRecord`